src/main/java/net/sf/jabref/event/AddEntryEvent.java

```diff
@@ -0,0 +1,18 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ * Instances of this class are suitable for passing through the event bus.
```

Comment:
The first comment is a bit superfluous. The second is more appropriate as a `@param` comment for the constructor. I would just keep the last line, i.e. `Event fired when a BibEntry was added to the db`